frontend/src/components/taste/TasteList.vue

```diff
@@ -57,30 +61,39 @@ export default {
             } else {
                 this.addTaste(id);
             }
-            tasteData.isClick = true;
-            tasteData.isAnimate = true;
-            setTimeout(function () {
-                tasteData.isAnimate = false;
-            }, 200);
+            if (this.allTastes.byId[id].is_default) {
+                tasteData.isClick = true;
+                tasteData.isAnimate = true;
+                setTimeout(function () {
+                    tasteData.isAnimate = false;
+                }, 200);
+            }
         },
         deleteTaste(id) {
-            this.tastesData[id].check = false;
-            this.$store.dispatch('taste/deleteMyTaste', id);
-            this.selectedIds.splice(this.selectedIds.indexOf(id), 1);
+            if (this.allTastes.byId[id].is_default) {
```

Comment:
Are you sure that by `id` will be found an object? If not it will throw an exception. It would be much safer if you save data to a variable and then will use it